newIDE/app/src/UI/SortableVirtualizedItemList/index.js

```diff
@@ -103,6 +103,7 @@ export default class SortableVirtualizedItemList<Item> extends React.Component<
           <ScreenTypeMeasurer>
             {screenType => (
               <List
+                style={{overflowX: 'hidden'}}

```

Comment:
@4ian I don't know why that horizontal scrollbar popped up. I guess this is a good safety check for future, since redundant scrollbars are bound to come around when, by defalut, overflow is set to auto. What do you suggest?

Reply:
I'd like to understand what is going out of the box and making the scrollbar to appear.
If needed, it should be added into `List` - in no cases a list should scroll horizontally, the content should always resize to the available width.

Reply:
> I'd like to understand what is going out of the box and making the scrollbar to appear.

@4ian I did a stupid mistake while merging :sweat_smile::joy: Everything is fine, I'm resolving this thread and pushing a new commit :+1:

Reply:
Great news!